lib/render/leaderboard.php

```diff
@@ -267,15 +267,17 @@ function RenderTopAchieversComponent($user, array $gameTopAchievers, array $game
     // Latest Masters Tab
     echo "<div id='latestmasters' class='tabcontentscores' style=\"display: " . ($numLatestMasters >= $masteryThreshold ? "block" : "none") . "\">";
     echo "<table class='smalltable'><tbody>";
-    echo "<tr><th>Pos</th><th colspan='2' style='max-width:30%'>User</th><th>Mastery Date</th></tr>";
+    echo "<tr><th>#</th><th>User</th><th>Mastered on</th></tr>";
```

Comment:
I think just "Mastered" would be enough. I planned on reducing most of those table headers to simpler terms a while ago.